src/test/java/sorinfoca/Tests/Tema8/ConfigPage/ConfigPages.java

```diff
@@ -0,0 +1,37 @@
+package sorinfoca.Tests.Tema8.ConfigPage;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+public class ConfigPages {
```

Comment:
Please rename this page to BaseTest and move it to same package with test classes.